include/ignition/rendering/Camera.hh

```diff
@@ -193,6 +202,24 @@ namespace ignition
       /// \return Camera view matrix
       public: virtual math::Matrix4d ViewMatrix() const = 0;
 
+      /// \brief Set the projection matrix for this camera
+      /// \param[in] _matrix Camera projection matrix
+      public: virtual void SetProjectionMatrix(const math::Matrix4d &_matrix)
+          = 0;
+
+      /// \brief Get the projection type for this camera
+      /// \return Camera projection type
+      public: virtual CameraProjectionType ProjectionType() const = 0;
+
+      /// \brief Set the projection type for this camera
+      /// \param[in] _matrix Camera projection type
+      public: virtual void SetProjectionType(CameraProjectionType _type) = 0;
```

Comment:
Should we document the relationship between projection type and projection matrix, and that when a type we (may?) ignore the custom projection matrix?

Reply:
added more doc on the behavior of calling this function. 6e2e25f